app/src/main/cpp/skyline/applet/swkbd/software_keyboard_applet.h

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/am/applet/IApplet.h>
+#include <services/applet/common_arguments.h>
+#include <jni.h>
+#include "software_keyboard_config.h"
+
+namespace skyline::applet::swkbd {
+    /**
+     * @brief Return values for swkbd
+     * @url https://switchbrew.org/wiki/Software_Keyboard#CloseResult
+     */
+    enum class CloseResult : u32 {
+        Enter = 0x0,
+        Cancel = 0x1,
+    };
+
+    /**
+     * @url https://switchbrew.org/wiki/Software_Keyboard#TextCheckResult
+     */
+    enum class TextCheckResult : u32 {
+        Success = 0x0,
+        ShowFailureDialog = 0x1,
+        ShowConfirmDialog = 0x2,
+    };
+
+    //The number of bytes used to put the text on the storages returned by swkbd
+    static constexpr u32 SwkbdTextBytes{0x7D4};
+
```

Comment:
Use doxygen style //!< Size of the returned IStorage buffer that's used to return the input text